services/sleeper/ci/gitlab-ci.yml

```diff
@@ -0,0 +1,160 @@
+image: itisfoundation/gitlab-runner-docker:stable
+variables:
+  # docker in docker variant, see https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#use-docker-in-docker-executor
+  DOCKER_HOST: tcp://docker:2375/
+  DOCKER_DRIVER: overlay2
+  # have some prefix so it may help for garbage collecting in the registry
+  SC_CI_PROJECT_PATH_NAME: ci/$CI_PROJECT_PATH_SLUG/$CI_COMMIT_REF_SLUG
+  SC_CI_TEST_IMAGE_PREFIX: $SC_CI_TESTING_REGISTRY/ci/$CI_PROJECT_PATH_SLUG/$CI_COMMIT_REF_SLUG
+  SC_CI_SLEEPER_LOCATION: models/sleeper
```

Comment:
is it models or services?

Reply:
I have no idea about the difference but I know it's a computational service

Reply:
:rofl: 
actually models is the archaic name for services... @mguidon 
but also has some other meaning I think...

anyway since the gitlab is not used here... I guess this is also wrong in case it would be in use. maybe just remove it from there.

Reply:
removed line 9